packages/com.pirate.refresh.yml

```
@@ -0,0 +1,20 @@
+title: Auto Dev Token Refresh
+iconUri: https://github.com/LucifersCircle/WebOS-Token-Refresh-App/blob/main/src/app/icon.png?raw=true
```

In case the icon is moved to some other places, I'd suggest following URL:

```suggestion
iconUri: https://raw.githubusercontent.com/LucifersCircle/WebOS-Token-Refresh-App/df2ddd0d429e86b024777f9be762e242e8eb1106/src/app/icon.png
```